gateway/api/v1/views/jobs.py

```diff
@@ -70,6 +70,13 @@ def provider_list(self, request):
     def result(self, request, pk=None):
         return super().result(request, pk)
 
+    @swagger_auto_schema(
+        operation_description="Update the sub status of a job",
```

Comment:
I'm not 100% sure but I think we need to add `sub_status` here, `swagger_generator` won't fill that info directly by itself.